xenit/build.gradle

```diff
@@ -1,20 +1,28 @@
-import com.bmuschko.gradle.docker.tasks.image.Dockerfile.From
 import com.avast.gradle.dockercompose.tasks.ComposeDown
+import com.bmuschko.gradle.docker.tasks.image.Dockerfile.From
 
 subprojects {
     apply plugin: 'eu.xenit.docker'
     apply plugin: 'eu.xenit.docker-compose'
 
     configurations {
         telemetry
+        actuators
     }
 
     ext {
         alfredTelemetryVersion = '0.5.0'
+        actuatorsVersion = '0.0.1'
+    }
+
+    dependencies {
+        actuators("eu.xenit.solr-actuators:solr-actuators:${actuatorsVersion}") {
+            transitive = false
```

Comment:
Not blocking for this PR, but if you need to use `transitive = false`, it indicates that something is wrong in the dependencies of the solr-actuators project.